test/conf/nginx.conf

```diff
@@ -153,6 +153,46 @@ http {
         }
     }
 
+    server {
+        listen 58089 ssl;
+        server_name localhost;
+        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers         HIGH:!aNULL:!MD5;
+        ssl_certificate     __NGXDOCROOT__/dummy.crt;
+        ssl_certificate_key __NGXDOCROOT__/dummy.key;
+        ssl_verify_client on;
+        ssl_client_certificate __NGXDOCROOT__/dummy.crt;
+
+        mruby_ssl_verify_client_handler_code 'Nginx::SSL.reject_client';
+    }
+
+    server {
+        listen 58090 ssl;
+        server_name localhost;
+        ssl_protocols       TLSv1 TLSv1.1 TLSv1.2;
+        ssl_ciphers         HIGH:!aNULL:!MD5;
+        ssl_certificate     __NGXDOCROOT__/dummy.crt;
+        ssl_certificate_key __NGXDOCROOT__/dummy.key;
+        ssl_verify_client on;
+        ssl_client_certificate __NGXDOCROOT__/dummy.crt;
+
+        mruby_ssl_verify_client_handler_code 'Nginx::SSL.accept_client';
```

Comment:
1) In this case, we should test that the certificate is invliad but accept the TLS session by `accept_client` method.